packages/components/src/theme/theme-spectrum/theme-spectrum-alias.module.css

```diff
@@ -1,6 +1,13 @@
 /* stylelint-disable custom-property-empty-line-before */
 /* stylelint-disable alpha-value-notation */
-.dh-spectrum-alias {
+
+/**
+ * The higher-alias-specificity class is so we always take precedence over spectrum's definitions 
+ * It must be different from any other CSS module specificity classes for spectrum's theme
+ * Otherwise one of the classes will not get applied since the JS part receives the same key,
+ * but the CSS module will generate a different class name suffix. This ends up with spectrum picking only 1.
+ */
+.dh-spectrum-alias.higher-alias-specificity {
```

Comment:
I would prefix the specificity class with `.dh-` for consistency. Alternative would be to just double the original class `.dh-spectrum-alias.dh-spectrum-alias` and then you don't need a new name. IMO makes it a little more clear you are just doubling the specificity not targetting a different thing.

Reply:
Neat. Didn't know that was valid in CSS to increase specificity. I'll change and make sure the CSS modules handles it properly (doesn't assign 2 separate suffixes for some reason)